pkg/core/serving/knative/servingrun.go

```diff
@@ -256,26 +255,22 @@ func (r *servingRun) createService(s *openfunction.Serving, cm map[string]string
 				case "min-scale", "minScale":
 					minScale = v
 				}
-				if _, exist := s.Spec.Annotations[fmt.Sprintf("%s/%s", knativeAutoscalingPrefix, k)]; !exist {
+				if _, exist := s.Spec.Annotations[fmt.Sprintf("autoscaling.knative.dev/%s", k)]; !exist {
```

Comment:
I mean we should put `autoscaling.knative.dev/min-scale: 3` in scaleoption.knative, not `min-scale: 3` to be consistent with knative.

The other options are similar.
so might be something like below?:
```go
				switch k {
				case "autoscaling.knative.dev/", "autoscaling.knative.dev/":
					maxScale = v
				case "autoscaling.knative.dev/", "autoscaling.knative.dev/":
					minScale = v
				}
```

Reply:
are they too long as non-annotation form configurations...

Reply:
user can simply copy knative annotation options here without translation might be better?

Reply:
this does reduce the comprehension cost for the user, while it duplicates the effects of `spec.serving.annotations `